### PR TITLE
Us-6 Moved Burnup Chart from Main Scene to Button Click new window

### PR DESCRIPTION
**Pull request by prosm056**
Screen Shot 2022-02-11 at 10 11 54 AM

### CODE REVIEW

File: ScrumBoardSimulator/src/main/java/se/bettercode/scrum/ScrumGameApplication.java [hunk 1]
```diff
@@ -34,10 +33,12 @@
     private Sprint sprint;
     private Team team;
     private Backlog backlog;
+    private String chart;
     private StatusBar statusBar = new StatusBar();
     private SelectableBacklogs backlogs = new SelectableBacklogs();
+    private SelectableCharts charts = new SelectableCharts();
```
Comment (Rephaelbj) [inline ScrumBoardSimulator/src/main/java/se/bettercode/scrum/ScrumGameApplication.java:39]: Missing, can't run

File: ScrumBoardSimulator/src/main/java/se/bettercode/scrum/ScrumGameApplication.java [hunk 2]
```diff
@@ -25,6 +22,8 @@
 import java.io.InputStreamReader;
 import java.util.Scanner;
 
+import java.awt.*;
```
Comment (Rephaelbj) [inline ScrumBoardSimulator/src/main/java/se/bettercode/scrum/ScrumGameApplication.java:25]: Remove, we are using javafx

File: ScrumBoardSimulator/src/main/java/se/bettercode/scrum/ScrumGameApplication.java [hunk 3]
```diff
@@ -107,7 +108,7 @@ private void setStage() {
         BorderPane borderPane = new BorderPane();
         board.prefWidthProperty().bind(primaryStage.widthProperty());
         borderPane.setCenter(board);
-        borderPane.setRight(burnupChart);
+//        borderPane.setRight(burnupChart);
```
Comment (Rephaelbj) [inline ScrumBoardSimulator/src/main/java/se/bettercode/scrum/ScrumGameApplication.java:111]: delete line

**Review by Rephaelbj — CHANGES_REQUESTED** [review]

File: ScrumBoardSimulator/src/main/java/se/bettercode/scrum/ScrumGameApplication.java (hunk 1 above)
Comment (prosm056) [reply ScrumBoardSimulator/src/main/java/se/bettercode/scrum/ScrumGameApplication.java]: Removed it

**Review by Rephaelbj — CHANGES_REQUESTED** [review]
When the burndown chart is open, nothing else can be clicked. This seems like an instance of future technical debt.

**Comment by prosm056** [comment]
> When the burndown chart is open, nothing else can be clicked. This seems like an instance of future technical debt.

So I did that on purpose for user to focus on the chart. It can change to make both work parallely.

**Review by Rephaelbj — APPROVED** [review]

**Pull request merged**